main/models/category.py

```diff
@@ -0,0 +1,9 @@
+from sqlalchemy import Column, Integer, ForeignKey, NVARCHAR, Text
+
+from main.models.base_class import Base
+
+
+class Category(Base):
+    name = Column(NVARCHAR(length=50), unique=True, nullable=False)
+    description = Column(Text(length=5000))
+    user_id = Column(Integer, ForeignKey("user.id"))
```

Comment:
Please also define the relationships between models as well. `Category.user, Item.category, Item.user...`